packages/run-protocol/src/psm/psm.js

```diff
@@ -67,31 +66,20 @@ export const start = async (zcf, privateArgs) => {
       anchorPerStable.denominator.brand === stableBrand,
     X`Ratio ${anchorPerStable} is not consistent with brands ${anchorBrand} and ${stableBrand}`,
   );
+  const bpRatio = bp => makeRatio(bp, stableBrand, BASIS_POINTS);
```

Comment:
oops; I forgot to tell you: @dtribble said basis points are too coarse; he wants the governance parameters to be ratios.

Reply:
We don't have a brand-less ratio. We could do a pair of nats to make one.

Is there a grain size that would be sufficient, like 10^-12?

Reply:
I don't see a need for brand-less ratios. The caller would have to supply the right brands. Am I confused?

Reply:
Cool. I was confusing this with another conversation in RUN meeting.

Done.